recipes/sherlockml-boltzmannclean/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sherlockml-boltzmannclean" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "0c028e9f8a67fac787587c20e24ef69792899d1c76e1313515518a9aff0f57ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
It does not appear as if `numpy` is required to actually build the package. If this is the case, please remove it from the `build` requirement list.